src/analyze/analyze.c

```diff
@@ -2145,7 +2179,56 @@ static int do_condition(int argc, char *argv[], void *userdata) {
 }
 
 static int do_verify(int argc, char *argv[], void *userdata) {
-        return verify_units(strv_skip(argv, 1), arg_scope, arg_man, arg_generators);
+        char **filename;
+        _cleanup_strv_free_ char **filenames = NULL;
+        _cleanup_(rm_rf_and_freep) char *tempdir = NULL;
+        bool setup_alias = false;
+
+        STRV_FOREACH(filename, strv_skip(argv, 1))
+                if (strchr(*filename, ':')) {
+                        setup_alias = true;
+                        break;
+                }
+
+        if (setup_alias) {
+                int r, i = 1;
+                char *dst;
+
+                r = setup_working_dir(&tempdir);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to setup working directory");
+
+                STRV_FOREACH(filename, strv_skip(argv, 1)) {
+                        if (strchr(*filename, ':')) {
+                                _cleanup_free_ char *src = NULL, *argname = NULL;
+                                struct stat statbuf;
+
+                                argname = strdup(*filename);
+                                r = extract_first_word((const char **)filename, &src, ":", 0);
+                                if (r < 0 || !*filename)
+                                        return log_error_errno(r, "Failed to parse alias %s", argname);
+
+                                dst = path_join(tempdir, basename(*filename));
+                                if (!dst)
+                                        return log_oom();
+
+                                r = stat(src, &statbuf);
+                                if (r < 0)
+                                        return log_error_errno(errno, "Couldn't stat %s", src);
```

Comment:
if there's an error here or below, `dst` will be leaked - declare it inside the loop as `_cleanup_free_`, and then use strv_consume(.. TAKE_PTR(dst))`